xxh3.h

```diff
@@ -1270,6 +1327,8 @@ XXH3_scrambleAcc(void* XXH_RESTRICT acc, const void* XXH_RESTRICT secret)
 }
 
 #define XXH_PREFETCH_DIST 384
+#define XXH_PREFETCH_DIST_AVX512_64  496
```

Comment:
I'm surprised this distance, 496, is not a multiple of 64.
What about 512 ? or 448 ?

My understanding is that, for a distance 496, and presuming a perfectly aligned input on 64-bytes boundaries, 496 is effectively equivalent to 448. Now, for the more general case of non-aligned input, 496 will sometimes be equivalent to 448, sometimes equivalent to 512, depending on the specific `input_address % 64` value.
And I'm not sure if this behavior is really a benefit.

Reply:
according to [this](https://www.cc.gatech.edu/~hyesoon/lee_taco12.pdf), the prefetch distance have to be profiled on the benchmark machine (2 core ECS on Aliyun, Cannon lake CPU). I tested from 128 to 512, it seems 496 is the best.

Reply:
> I tested from 128 to 512, it seems 496 is the best.

Well, I have no access to the program you used to measure this value.
But I would expect the outcome of this program to be subtly impacted by input's alignment.
I would actually expect this input to feature `start_address % 64 == 16`.

Try multiple different `start_address`.

Reply:
ok, i will try this today.